### ISSUE TITLE
Setup Products Dynamic Links

### ISSUE THREAD

**Issue by himanshusharma89** [himanshusharma89/relic_bazaar#51]
Follow up of #50 
Use dynamic links to launch the product screen(directly) when the product is shared by someone.
If the app is not installed lead to the Play/App Store.

**Comment by sssash18** [comment]
Can I work on this issue @himanshusharma89 ? But it would be only possible after issue #50  is resolved right ?

**Comment by himanshusharma89** [comment]
Yeah, we need to work on #50 to resolve this and also one person can be assigned to only one issue at a time.

**Comment by subhangi2731** [comment]
@himanshusharma89 please assign this issue to me
